internal/aws/awsutil/conn.go

```diff
@@ -70,11 +71,16 @@ const (
 
 // newHTTPClient returns new HTTP client instance with provided configuration.
 func newHTTPClient(logger *zap.Logger, maxIdle int, requestTimeout int, noVerify bool,
-	proxyAddress string) (*http.Client, error) {
+	proxyAddress string, certificateFilePath string) (*http.Client, error) {
 	logger.Debug("Using proxy address: ",
 		zap.String("proxyAddr", proxyAddress),
 	)
-	tls := &tls.Config{
+	rootCA, err := loadCertPool(certificateFilePath)
+	if certificateFilePath == "" && err != nil {
+		logger.Warn("could not create root ca from", zap.String("file", certificateFilePath), zap.Error(err))
+	}
+	tlsConfig := &tls.Config{
```

Comment:
Adding pem to tls certificates field does not work. You have to add them to rootca field. I should have spent more time reading the docs then seeing a field that looked like the right field. I spent way to much time wondering why this was not working.